src/Core/Content/Maker/DependencyInjection/CompilerPass/CreateGeneretorScafoldingCommandPass.php

```diff
@@ -0,0 +1,66 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Content\Maker\DependencyInjection\CompilerPass;
+
+use Shopware\Core\Content\Maker\Command\MakerCommand;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Plugin\Command\Scaffolding\Generator\ScaffoldingGenerator;
+use Symfony\Component\DependencyInjection\ChildDefinition;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+#[Package('content')]
+class CreateGeneretorScafoldingCommandPass implements CompilerPassInterface
+{
+    public function process(ContainerBuilder $container): void
```

Comment:
can you also add a unit test for the compiler pass? You should be able to create a dummy container from scratch that is properly set up for the cases you want to test and then run this process method and check that the commands are registered as expected in the container

Reply:
@keulinho, done